src/features/terminal/terminalManager.ts

```diff
@@ -369,6 +395,12 @@ export class TerminalManagerImpl implements TerminalManager {
     }
 
     public async initialize(api: PythonEnvironmentApi): Promise<void> {
+        for (const t of terminals()) {
```

Comment:
@karthiknadig confirming it makes sense to go through all the open terminals to add these. my two questions:
- confirming this is desired behavior
- checking if there is a better way to do it (/ if I should rework the function) to be more efficient